src/types/api/query.ts

```diff
@@ -63,6 +63,7 @@ export type ExecuteYdbResponse = {
     result: KeyValueRow[];
 } & CommonFields;
 
+// prettier-ignore
```

Comment:
We ignore them because when used with TS conditions it makes them flat and less readable
```
type ExecuteResponse<Schema extends Schemas> =
    | CommonFields // result can be undefined for queries like `insert into`
    | (Schema extends 'modern'
          ? ExecuteModernResponse
          : Schema extends 'ydb'
            ? ExecuteYdbResponse
            : Schema extends 'classic' | undefined
                ? ExecuteClassicResponse
                : unknown);
```

```
type ExecuteResponse<Schema extends Schemas> =
    | CommonFields // result can be undefined for queries like `insert into`
    | (Schema extends 'modern'
          ? ExecuteModernResponse
          : Schema extends 'ydb'
          ? ExecuteYdbResponse
          : Schema extends 'classic' | undefined
          ? ExecuteClassicResponse
          : unknown);
```